requirements/requirements-flash_attn.txt

```diff
@@ -0,0 +1 @@
+pip install flash-attn --no-build-isolation
```

Comment:
Do we need `--no-build-isolation` here?  If not, we could just list `flash-attn` here?

Reply:
I added because the official document shows the command with `--no-build-isolation` but am not sure if we really need it. Let me try to remove it now.